std/math.d

```diff
@@ -4270,10 +4270,13 @@ struct IeeeFlags
 private:
     // The x87 FPU status register is 16 bits.
     // The Pentium SSE2 status register is 32 bits.
+    // PowerPC FPSCR is a 32-bit register.
+    // ARM FPSCR is a 32bit register
+    // SPARC FSR is a 32bit register (64 bits for SPARC 7 & 8, but high bits are uninteresting).
     uint flags;
+
     version (X86_Any)
```

Comment:
Can we move `CRuntime_Microsoft` to druntime? And if necessary add `version (X86_Any)` and `version (ARM)` inside.